build/buildx/multiarch-images.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Building multi-platform images
+description: Different strategies for building multiplatform images
+keywords: build, buildx, buildkit, multiplatform images
+---
```

Comment:
Why it's there?

Reply:
@crazy-max Looks like other commits were accidentally included in this PR. We need to cherry pick just the Compose release notes commit.

Reply:
@dockertopia Not sure if these were intentionally included in this PR. If so, suggest we remove commits other than Compose release notes and move those to a separate PR.

Reply:
Yes it was unintentional. I understood after resolving conflicts previous the state of Build had been preserved. But if not, it needs fixing. If it can wait I can check this on Monday otherwise feel free to fix it @usha-mandya or assign it to someone else, whatever is best.